### PR TITLE
a2dismod to avahi-browse: add Chinese translation

### PR DESCRIPTION
**Pull request by zjuyk**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by zjuyk** [comment]
Add translation for commands which begin with 'a'.

**Comment by bl-ue** [comment]
@zjuyk several pages have a space after the Chinese colon&mdash;will you please remove them?  If there's a Chinese colon, it shouldn't have a space after. Only English colons need spaces after.

**Comment by zjuyk** [comment]
I'll review and fix them.

**Comment by zjuyk** [comment]
@bl-ue I remove the space after the Chinese colon. Are there any other typos?

**Comment by bl-ue** [comment]
Yes, these pages have more than one newline:

- a2enconf
- aplay
- arecored

Please make sure they have exactly 1 newline at the end.

File: pages.zh/linux/add-apt-repository.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 > apt 仓库管理.
```
Comment (bl-ue) [inline pages.zh/linux/add-apt-repository.md:3]:
```suggestion
> apt 仓库管理。
```

File: pages.zh/linux/arithmetic.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- 开始算术测试：
+
+`arithmetic`
+
+- 指定一个或多个算术运算符来设计问题：
+
+`arithmetic -o {{+|-|x|/}}`
+
+- 指定范围。加法和乘法问题限定 0 到指定范围之间的数字，包含上区间。减法和除法问题限制结果和运算数字必须在 0 到指定范围之间。
```
Comment (bl-ue) [inline pages.zh/linux/arithmetic.md:14]:
```suggestion
- 指定范围。加法和乘法问题限定 0 到指定范围之间的数字，包含上区间。减法和除法问题限制结果和运算数字必须在 0 到指定范围之间：
```

File: pages.zh/linux/ac.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# ac
+
+> 打印用户连接时长数据
```
Comment (bl-ue) [inline pages.zh/linux/ac.md:3]:
```suggestion
> 打印用户连接时长数据。
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi, I found a few oversights.

**Comment by bl-ue** [comment]
Well this looks good @zjuyk! Hopefully #5240 will agree with this, because the style you're using in these pages (pure Chinese punctuation) is the best, I think.  But I'd like to wait for that one to merged, then I'll merge this one.

**Review by Proscream — APPROVED** [review]

**Comment by bl-ue** [comment]
I marked this PR as a draft to avoid merging.  It's waiting on #5240.

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by bl-ue** [comment]
Waiting for #5240.  Sorry, I need to get back to that.

**Comment by bl-ue** [comment]
I'll refresh this now.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by bl-ue** [comment]
Apologies @zjuyk for this 可怕的传奇！😅  I'll merge this PR now, _finally_.

**Pull request merged**